pkg/tagger/utils/expiry_test.go

```diff
@@ -0,0 +1,75 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package utils
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestUpdate(t *testing.T) {
+	expire := &Expire{
+		expiryDuration: 5 * time.Minute,
+		lastSeen:       make(map[string]time.Time),
+	}
+	testContainerID := "b2beae57bb2ada35e083c78029fe6a742848ff021d839107e2ede87a9ce7bf50"
+
+	now := time.Now()
+	twoMinutesAgo := now.Add(-2 * time.Minute)
+
+	// Inserting container, we expect True as it's not in the lastseen map.
+	found := expire.Update(testContainerID, twoMinutesAgo)
+	require.True(t, found)
+	require.Len(t, expire.lastSeen, 1)
```

Comment:
if checking the lenght, check it's zero before inserting, just in case

Reply:
👌

Reply:
Add a `require.Len(t, expire.lastSeen, 0)` before calling Update the first time

Reply:
Right right, I only added it in the Expire: https://github.com/DataDog/datadog-agent/blob/957d37586fb8a17123d3f2d2a6f8c7e3ace3e4ce/pkg/tagger/utils/expiry_test.go#L53
Updating.